src/services/nodeVersion.json

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Can you document where you got this file? Maybe in a comment in `functionApp.ts`?